src/Persistence.LinnApps/Repositories/StockTriggerLevelsRepository.cs

```diff
@@ -33,5 +33,20 @@ public IQueryable<StockTriggerLevel> FindAll()
         {
             throw new NotImplementedException();
         }
+
+        public StockTriggerLevel FindById(int key)
+        {
+            throw new NotImplementedException();
```

Comment:
Does this work when you run it? These are all being used by the facade service so will need to be implemented I expect.